server/schedulers/balance_region.go

```diff
@@ -155,16 +157,16 @@ func (s *balanceRegionScheduler) transferPeer(cluster schedule.Cluster, region *
 	}
 
 	target := cluster.GetStore(storeID)
-	log.Debugf("[region %d] source store id is %v, target store id is %v", region.GetID(), source.GetID(), target.GetID())
+	log.L().Debug("", zap.Uint64("region", region.GetID()), zap.Uint64("source-store", source.GetID()), zap.Uint64("target-store", target.GetID()))
```

Comment:
It seems that we need to unify the naming among all PRs. Here the kebab style is used, but in #1422 space-separated strings are used, see https://github.com/pingcap/pd/pull/1422/files#diff-44da3d359afd2823a31e4fb048bdfdf4R58
/cc @nolouch